src/main/javascript/acd/ol/ol.mapwithactions.js

```diff
@@ -1,7 +1,14 @@
 acd.ol.MapWithActions = function (options) {
 	var self = this;
 	this.actions = [];
-	
+
+	if (options.disableRotation) {
+		if(!options.interactions){
+			options.interactions = ol.interaction.defaults(
+					{altShiftDragRotate: false, pinchRotate: false});
+		}
+	}
```

Comment:
Code wijziging hiervoor zit in #2 dus zorgt ervoor dat deze merge niet kan gebeuren, of voor #2 moet gebeuren. Best de wijzigingen hier uitvoeren.